shell/common/thread_host.cc

```diff
@@ -35,6 +39,7 @@ void ThreadHost::Reset() {
   ui_thread.reset();
   raster_thread.reset();
   io_thread.reset();
+  profiler_thread.reset();
```

Comment:
@chinmaygarde I agree that there only needs to be one profiler for all the shells. Moving this to the concurrent thread loop is a more involved change, given that the concurrent thread loops don't have scheduling mechanisms (at fixed delays).

Mind if I file an issue for this and address it in a follow up change. I think this will be useful as-is.

Reply:
Sounds good.